ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/manage/[ingestKeys]/[keyID]/TransformEvent.tsx

```diff
@@ -53,7 +53,7 @@ export function createTransform({
 }): string {
   return `// transform accepts the incoming JSON payload from your
 // webhook and must return an object that is in the Inngest event format
-function transform(evt, headers = {}) {
+function transform(evt, headers = {}, queries = {}) {
```

Comment:
What about calling this `queryParams` by default? The user can change if they'd like it, but I think "params" is a more ubiquitous term than "queries."
```suggestion
function transform(evt, headers = {}, queryParams = {}) {
```